themes/default/data/registry/packages/self-signed-cert.yaml

```
@@ -0,0 +1,14 @@
+category: Cloud
+component: true
+description: A Pulumi component that creates a self-signed certificate.
+featured: false
+logo_url: ""
+name: self-signed-cert
```

```suggestion
name: tls-self-signed-cert
```
this is causing the "view source code" link to 404 since it is looking for a different repo name

---

Ah good catch, thanks!!